financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -97,6 +100,28 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
     });
 
     return new MedianizerPriceFeed(priceFeeds);
+  } else if (config.type === "balancer") {
+    const requiredFields = ["balancerAddress", "balancerTokenIn", "balancerTokenOut"];
```

Comment:
Just trying to understand this better, will the `balancerAddress` be the address of the balancer "Pool" that we are interested in querying an exchange rate for? Where the pool has a tokenIn and a tokenOut, and the current price is the exchange rate between them?

Reply:
yes you are right, `tokenIn`, `tokenOut` is just how they label the pair for the price ratio you want